setup.py

```diff
@@ -11,4 +11,13 @@
 with open("requirements.txt") as handle:
     REQUIRES = list(handle.read().splitlines())
 
-setup(install_requires=REQUIRES)
+setup(
+    name="weblate_schemas",
+    entry_points={
+        "fedora.messages": [
+            "base.message=weblate_schemas.messages:BaseMessage",
```

Comment:
Please prefix the key here with `weblate.`. The `base.message` key is already used by Fedora Messaging